0000-template.md

```diff
@@ -2,7 +2,7 @@
 - Title: [edit and replace with a unique name like so: `my-awesome-proposal`]
 - Authors: [Your Name](mailto:your@email.com)
 - Start date: [edit and replace the day work begun, i.e. `Jan 15, 2019`]
-- RFC PR: Edit if merged: [mimblewimble/grin-rfcs#0000](https://github.com/mimblewimble/grin-rfcs/pull/0000)
+- RFC PR: Edit if merged: [mimblewimble/grin-rfcs#0000](https://github.com/mimblewimble/grin-rfcs/pulls/0000)
```

Comment:
I understand the change is for the CI. But I think this might led to confusions in the future.